src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx

```diff
@@ -86,24 +84,22 @@ class ReleaseProgress extends AsyncView {
 
   handleClick(action) {
     let {project, organization} = this.context;
-    this.api.request('/promptsactivity/', {
-      method: 'PUT',
-      data: {
-        organization_id: organization.id,
-        project_id: project.id,
-        feature: 'releases',
-        status: action,
-      },
-      success: data => {
-        this.setState({showBar: false});
-      },
-    });
+
+    let params = {
+      projectId: project.id,
+      organizationId: organization.id,
+      feature: 'releases',
+      status: action,
+    };
+    promptsUpdate(this.api, params).then(this.setState({showBar: false}));
```

Comment:
:+1:

Reply:
Thanks for your help debugging this Mark!